js/index.js

```diff
@@ -635,6 +659,20 @@ $(function() {
 
                           myconsole.warn("Unable to load JS bundle", err);
                         });
+                      } else if ($('input[name=backend_inputs]').filter(':checked').val() === "mathbox") {
+                           $.when($.get("js/mathbox-bundle.js"),
+                                  $.get(backend.endpoint + "/bundle")
+                                ).done(function(mathboxBundle, bundle) {
+
+                           var replaced = bundle[0].replace(/require\("Data.Either"\)/g, "PS['Data.Either']")
+                                                   .replace(/require\("Data.Maybe"\)/g, "PS['Data.Maybe']")
+                                                   .replace(/require\("Mathbox.Field"\)/g, "PS['Mathbox.Field']")
```

Comment:
@paf31 ok, I managed to get rid of them and published a new version: https://pursuit.purescript.org/packages/purescript-mathbox/0.1.1

If you can make it available at the server side, then I will cleanup the code here.

BTW Wouldn't it be better if `psc-bundle` handled it the same as `pulp browserify`?